sdk/core/azure-core/src/http/curl/curl.cpp

```diff
@@ -17,6 +18,18 @@
 #include <thread>
 
 namespace {
+// Can be used from anywhere a little simpler
+inline void LogThis(std::string const& msg)
+{
+  if (Azure::Core::Logging::Details::ShouldWrite(
+          Azure::Core::Http::LogClassification::HttpTransportAdapter))
+  {
+    Azure::Core::Logging::Details::Write(
+        Azure::Core::Http::LogClassification::HttpTransportAdapter,
+        "[CURL Transport Adapter]: " + msg);
```

Comment:
I don't think that in the message you should be, basically, duplicating classification.